README.md

```diff
@@ -114,77 +121,77 @@ POST https://api.globalping.io/v1/measurements
 
-It supports a human friendly interface for issuing commands. To begin simply type `/globalping help`.
+To initiate a command with the Globalping Slack app, use the slash command `/globalping`. Apart from that, you can use the same test command structure as with our other tools and integrations. For example, to get started, type `/globalping help`.
 
-Examples:
+Example commands to try:
 ```
-/globalping {command} {target} from {location} --limit {number of probes}
+
 /globalping ping 8.8.8.8 from Germany
+
 /globalping traceroute jsdelivr.com from South America --limit 2
-/globalping help
+
 ```
 
-The location field can process all kinds of different types of location matching, including continents, regions, countries, cities, US states and ASNs. ASNs must be prefixed by "AS", e.g. `from AS80085`. 
-You can also combine filters, e.g. `from hetzner+Finland` will ensure the results come from a probe that matches both parameters.
-Providing no location will default to "world" which will match a probe from a random location in the world.
+>[!Important]
+> To make the app post test results in Slack Threads, mention it using `@globalping`. For example,  `@globalping ping google.com`
 
-You can even mention it in threads using `@globalping ping google.com`
+Learn more about the Slack app [on our blog](https://www.jsdelivr.com/blog/network-troubleshooting-in-slack-made-easy-with-globalping/).
 
 ### GitHub Bot
+Automatically post network test results into any **public** GitHub issue with our GitHub bot. Mention it with `@globalping`, followed by the command you want to run. Otherwise, you can follow the same test command structure as with our other tools and integrations.
 
-Our GitHub bot can by triggered by simply mentioning it in any public GitHub issue. It supports a human friendly format of issuing commands and follows the existing Slack and CLI formatting logic allowing you to easily move between integrations. 
-
-Examples:
+Example commands to try:
 ```
 @globalping ping 8.8.8.8 from Germany
 @globalping traceroute jsdelivr.com from South America --limit 2
 ```
 
-The location field can process all kinds of different types of location matching, including continents, regions, countries, cities, US states and ASNs. ASNs must be prefixed by "AS", e.g. `from AS80085`.
-
-## Best practices and Tips
-
-Learn to use Globalping in the most optimal way! All examples below are based on our CLI tool for simplicity.
+Learn more about the GitHub bot [on our blog](https://dev.to/globalping/supercharge-your-gitops-workflows-with-the-globalping-github-bot-341a).
 
-### Magic field
-All of our integrations are using the same API and our "magic" field as location input. 
+## How to write networking tests with Globalping
```

Comment:
I think we need a better title. Again this makes me think of automating some kind of network tests, because thats when you "write" network tests, in this case we "run" them.

Something like "Network measurements and tests"?

Reply:
I just re-read it and I am not sure how useful the whole section is. Maybe we can shorten it? All it says is that you need to check the details of the specific tool that you're using and repeats some basics. e.g. initiate command not needed, location should have a link to the "best practices" section, flags are too specific on the tool and test being used...